docs/developer-docs/smart-contracts/advanced-features/serving-http-request.mdx

```diff
@@ -160,13 +170,13 @@ export default Canister({
             headers: [];
             status_code:404;
         }
-    })
+    }
 });
 ```
 
-To learn more about serving an HTTP request in Typescript, refer to [the Azle book reference on incoming HTTP requests](https://demergent-labs.github.io/azle/http.html).
+To learn more about serving an HTTP request in Typescript, refer to [the Azle book reference on fetch](https://demergent-labs.github.io/azle/fetch.html).
```

Comment:
We wouldn't want to show devs fetch here if my previous comment holds, as DFINITY hasn't really introduced this concept anywhere else.

Though I think DFINITY maybe should show HTTP Server vs Candid RPC canisters, these concepts can apply also somewhat to Motoko and the Rust CDK.

Reply:
removed info about HTTP server